sparse/bm25/bm25-vector-generation.ipynb

```diff
@@ -0,0 +1,430 @@
+{
```

Comment:
<p>We need to load the original public available qoura dataset. This dataset is coming with pairs of qoura questions and a label, we need to add here code that flatten it to single question per row.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>